src/modules/template/router.js

```diff
@@ -24,4 +24,15 @@ module.exports = async (server, opts) => {
       res.status(500).send()
     }
   })
+
+  server.put('/templates/targets/:id/cssSelector', { schema: schema.updateTarget }, async (req, res) => {
```

Comment:
```suggestion
  server.put('/templates/targets/:id/cssSelector', { schema: schema.updateTargetCssSelector }, async (req, res) => {
```
Nó chỉ update css selector thôi mà, đúng chứ?